odoo/fields.py

```diff
@@ -2792,13 +2795,77 @@ def _record_ids_per_res_model(self, records):
         return model_ids
 
 
+class Command(enum.IntEnum):
+    """
+    :class:`~odoo.fields.One2many` and :class:`~odoo.fields.Many2many` fields
+    expect a special command to manipulate the relation they implement.
+    Internally, each command is a tuple where the first element is a mandatory
+    integer that identifies the command, the two next elements are arguments
+    related to the command.
+
+    Please refer to the :meth:`~odoo.models.BaseModel.write` documentation for
+    full details on the behavior of each command.
```

Comment:
Or maybe `Command` should be the reference, assuming Sphinx can properly render documentation for enum variants?